bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusRollershutterModuleHandler.java

```diff
@@ -44,6 +47,9 @@
     private final Logger logger = LoggerFactory.getLogger(NikobusRollershutterModuleHandler.class);
     private final List<PositionEstimator> positionEstimators = new CopyOnWriteArrayList<>();
 
+    private final Map<String, DirectionConfiguration> directionConfigurations = Collections
+            .synchronizedMap(new HashMap<>());
```

Comment:
Any reason this needs to be synchronized? Why not use a ConcurrentHashMap instead?